src/components/utils/DEPagination.js

```diff
@@ -0,0 +1,166 @@
+                    color="primary"
+                    size="small"
+                    ref={anchorRef}
+                    aria-controls={open ? menuId : undefined}
+                    aria-expanded={open ? "true" : undefined}
+                    aria-label={formatMessage(intl, "selectPageSize")}
+                    aria-haspopup="menu"
+                    onClick={handleToggle}
+                >
+                    {selectedPageSize} <ArrowDropDownIcon />
+                </Button>
+            </Tooltip>
+            <Popper
+                open={open}
+                anchorEl={anchorRef.current}
+                role={undefined}
+                transition
+                disablePortal
+            >
+                {({ TransitionProps, placement }) => (
+                    <Grow
+                        {...TransitionProps}
+                        style={{
+                            transformOrigin:
+                                placement === "bottom"
+                                    ? "center top"
+                                    : "center bottom",
+                        }}
+                    >
+                        <Paper>
+                            <ClickAwayListener onClickAway={handleClose}>
+                                <MenuList id={menuId}>
+                                    {options.map((option, index) => (
+                                        <MenuItem
+                                            key={option}
+                                            selected={
+                                                option === selectedPageSize
+                                            }
+                                            onClick={(event) =>
+                                                handleMenuItemClick(
+                                                    event,
+                                                    index
+                                                )
+                                            }
+                                        >
+                                            {option}
+                                        </MenuItem>
+                                    ))}
+                                </MenuList>
+                            </ClickAwayListener>
+                        </Paper>
+                    </Grow>
+                )}
+            </Popper>
+        </>
+    );
+}
+
+function DEPagination(props) {
+    const classes = useStyles();
+    const theme = useTheme();
+    const {
+        onChange,
+        page,
+        totalPages,
+        onPageSizeChange,
+        pageSize,
+        baseId,
+        intl,
+    } = props;
+    const matches = useMediaQuery(theme.breakpoints.down("sm"));
+    return (
+        <Paper className={classes.paper}>
+            <Grid container>
+                <Grid item sm={4} md={5}></Grid>
+                <Grid item>
+                    <Pagination
```

Comment:
Not for this PR, but eventually it would be nice to include a number field so the user can manually enter their own page number (it doesn't look like this MUI-labs component supports that out-of-the-box).

Maybe only if the number of pages exceeds a certain threshold?

Reply:
We can definitely can add that large screens at least.